dashboard/test/lib/services/script_seed_test.rb

```diff
@@ -11,6 +11,10 @@
 #
 # Also add a new test case which tests adding, creating, and deleting your new model.
 class ScriptSeedTest < ActiveSupport::TestCase
+  setup do
+    Game.game_cache = nil
```

Comment:
I had to add this because otherwise the `Game.by_name` cache would get populated for one test but never updated thereafter, causing incorrect results and test failures.